packages/documentation/src/stories/components/back-to-top/back-to-top.stories.ts

```diff
@@ -0,0 +1,55 @@
+import { StoryContext, StoryFn, StoryObj } from '@storybook/web-components';
+import { MetaComponent } from '@root/types';
+import { html } from 'lit';
+
+const meta: MetaComponent = {
+  id: '1a1b4cab-d0a8-4b01-bd85-b70e18668cb5',
+  title: 'Components/Back To Top Button',
```

Comment:
Having 'Button' before will help group all the buttons in the navigation (Also, I’m not sure why I removed the hyphens in my previous comment...)
```suggestion
  title: 'Components/Button Back-to-Top',
```